src/main/java/io/openshift/launchpad/ui/booster/ProjectInfoStep.java

```diff
@@ -115,7 +115,10 @@ public void initializeUI(UIBuilder builder) throws Exception
             Set<Version> versions = catalogService.getVersions(mission, runtime);
             if (versions != null && !versions.isEmpty()) {
                runtimeVersion.setValueChoices(versions);
-               runtimeVersion.setItemLabelConverter(Version::getName);
+               runtimeVersion.setItemLabelConverter((v) -> {
+                   Booster booster = catalogService.getBooster(mission, runtime, v).get();
+                   return booster.runtimeVersionName(v.getKey());
```

Comment:
Because Versions don't know their name. They're not obtained from the metadata.json like Runtimes and Missions, but they come from the Booster's own yaml file. (We talked about that yesterday, or the day before...)